mastracode/src/web/ui/styles.css

```
@@ -0,0 +1,1923 @@
+  --bg-surface: #101218;
+  --bg-surface2: #161922;
+  --bg-surface3: #1d212c;
+  --bg-elevated: #1a1d27;
+  --fg: #e8eaf0;
+  --fg-dim: #9aa1b2;
+  --fg-muted: #5a6072;
+  /* MastraCode identity: indigo→violet accent. */
+  --accent: #8b7bf7;
+  --accent-2: #b18bff;
+  --accent-hover: #a594ff;
+  --accent-soft: rgba(139, 123, 247, 0.14);
```

Stylelint (17.13.0)

[error] Expected "0.14" to be "14%" (alpha-value-notation)

(alpha-value-notation)